user/urls.py

```diff
@@ -1,9 +1,16 @@
 from django.urls import path
 
-from user.views import login_view, logout_view, auth_view
+from user.views import *
 
 urlpatterns = [
     path('login/', login_view),
     path('logout/', logout_view),
-    path('auth/', auth_view)
+    path('auth/', auth_view),
+    path('user/update', update_user_view),
+    path('allusers', list_users_view),
+    path('createuser', create_user_view),
```

Comment:
We can replace a lot of these with a router tied to that ViewSet I mentioned below this.

Reply:
9b4bbca2